dist/index.js

```diff
@@ -92,6 +92,14 @@ function getPrNumber() {
     }
     return pullRequest.number;
 }
+function getBranchName() {
```

Comment:
I think we can get the branch name using `github.head_ref`

https://docs.github.com/en/actions/learn-github-actions/contexts#github-context